RELEASE-VERSION

```diff
@@ -0,0 +1 @@
+0.10.1-14-g4f62380-dirty
```

Comment:
😏

Reply:
[the gist that `version.py` is based on](https://gist.github.com/jpmens/6248478) recommends adding this file to .gitignore (whilst including it in sdist tarballs so its available for users). Presumably this is to avoid having this file dirty when doing development, if `version.get_git_version` is called (I guess in setup.py or more likely via make).

Is there a reason why you include this file here?

Reply:
^ this

Reply:
Nope, good catch!